src/event_sub.rs

```diff
@@ -18,20 +17,29 @@ use crate::event_sub::EventSubError::*;
 use crate::tm_client::{EventData, EventType, TmClient};
 
 #[derive(Clone, Debug)]
-pub struct Event {
-    pub event_type: String,
-    pub attributes: HashMap<String, String>,
+pub enum Event {
+    Block(block::Height),
```

Comment:
```suggestion
    BlockStart(block::Height),
```
rename so it's clear if it is the start or end?